internal/statestore/backfill.go

```diff
@@ -280,22 +281,41 @@ func (rb *redisBackend) DeleteBackfillCompletely(ctx context.Context, id string)
 	return nil
 }
 
+func (rb *redisBackend) cleanupWorker(ctx context.Context, backfillIDsCh <-chan string, wg *sync.WaitGroup) {
+	var err error
+	for id := range backfillIDsCh {
```

Comment:
I think no need to add the type into the name here:
```suggestion
	for id := range backfillIDs {
```

Reply:
Seems that this is minor I will remove